src/style.css

```diff
@@ -1 +1,75 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: normal;
+  font-style: normal;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.big__input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  outline: none;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: 25px;
+  border: 1px solid rgba(225, 231, 237, 1);
+  box-shadow: 0 1 8 0 rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You should include a generic font family as a fallback. For example: `font-family: Avenir, Arial, sans-serif;`.